angr/knowledge_plugins/sync/sync_manager.py

```diff
@@ -129,6 +135,30 @@ def push_comments(self, comments):
             r |= self._client.get_state().set_comment(addr, comment)
         return r
 
+    @init_checker
+    def push_stack_variables(self, stack_variables: Set[SimStackVariable], var_manager: VariableManagerInternal):
+        """
+
+        :param stack_variables:
+        :param var_manager:
+        :return:
+        """
+        r = False
+        for var in stack_variables:
+            try:
+                var_type = str(var_manager.get_variable_type(var))
+            except:
```

Comment:
Please specify the type of exceptions you expect, or `except Exception` if you want to capture all Python exceptions (but this is not recommended).

Reply:
I was afraid that if any error happened here, analysis way down the line will get messed up hard. For now, it seems like only a KeyError can happen when a variable is looked up for its type. So for now, I will go with `except KeyError`.